Dockerfile

```diff
@@ -23,7 +23,7 @@ ENV PATH="$PATH_PYTHON3" VIRTUAL_ENV="$VIRTUAL_ENV_PYTHON3"
 
 # Install Python dependencies
 COPY requirements.txt /tmp/install/requirements.txt
-RUN install_pinned.py -r /tmp/install/requirements.txt
```

Comment:
Is install_pinned.py something on the docker base image?

Reply:
Yes. It handles an extended form of requirements file that allows another file to be imported as default versions, and it checks that every package has a version pin.

Reply:
It's totally sweet and awesome Merriness. Its pinning is better than Poetry :-P